Detectors/EMCAL/simulation/src/Digitizer.cxx

```diff
@@ -196,14 +204,12 @@ void Digitizer::setEventTime(double t)
 
   mEventTime = t - mTriggerTime;
 
+  mDigits.forwardMarker(mEventTime);
```

Comment:
Here I am not sure how it works, the intention was that the timing in the DigitsWriteoutBuffer is absolute, but this line makes it relative (to mTriggerTime), changing mTriggerTime you might get a time that is smaller than the current marker in the DigitsWriteoutBuffer